Makefile

```diff
@@ -277,3 +277,9 @@ HELM=helm-not-found
 else
 HELM=$(shell which helm)
 endif
+
+# find or download goreleaser
+goreleaser:
```

Comment:
Can you use the same way to install tools than for kubebuilder for example?

```make
goreleaser:
ifeq (, $(shell which goreleaser))
	# https://kubebuilder.io/quick-start.html#installation
	curl -sfL https://install.goreleaser.com/github.com/goreleaser/goreleaser.sh \
		| sh -s v0.129.0
	mv ./bin/goreleaser "$(GORELEASER)"
GORELEASER=$(GOBIN)/kubebuilder
else
GORELEASER=$(shell which goreleaser)
endif
```

Reply:
Then it would be nice to have a target `release` or `gorelease`.
```make
release:
	$(GORELEASER) ...
```

Reply:
fixed